docs/UserGuide.adoc

```diff
@@ -342,6 +342,22 @@ Alias: `h`
 Pressing the kbd:[&uarr;] and kbd:[&darr;] arrows will display the previous and next input respectively in the command box.
 ====
 
+=== Changing the theme : `theme` `since v1.3`
+
+Changes the theme of the application. +
+Format: `theme` +
+Alias: `t`
+
```

Comment:
Should we include a keyword (e.g. THEME_TYPE) at format and alias to indicate that user has to write an additional parameter to change theme?

Reply:
Oh yeah, I'll update that